### PR TITLE
Re-grouping and re-ordering go imports

### PR DESCRIPTION
**Pull request by coreydaley**
Re-grouping and re-ordering go imports according to our team preferences:
 - standard
 - other
 - kubernetes
 - openshift
 - module

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -6,18 +6,19 @@
 package tools
 
 import (
-	// Makefile
 	"github.com/jteeuwen/go-bindata/go-bindata"
-	"github.com/openshift/library-go/alpha-build-machinery/make/targets/openshift"
+	// Makefile
```
Comment (dmage) [inline tools.go]: this one is better to keep as is

Comment (coreydaley) [reply tools.go]: Fixed, good catch.  I need to update the script to preserve comments, or allow an exclude flag

**Pull request merged**